QgisModelBaker/libqgsprojectgen/dbconnector/mssql_connector.py

```diff
@@ -818,3 +818,19 @@ def create_basket(self, dataset_tid, topic):
                     'Could not create basket for topic "{}": {}'
                 ).format(topic, error_message)
         return False, self.tr('Could not create basket for topic "{}".').format(topic)
+
+    def get_tid_handling(self):
+        if self.schema and self._table_exists(SETTINGS_TABLE):
+            cur = self.conn.cursor()
+            cur.execute(
+                """SELECT setting
+                            FROM {schema}.{table}
+                            WHERE tag = 'ch.ehi.ili2db.TidHandling'
+                            """.format(
+                    schema=self.schema, table=SETTINGS_TABLE
```

Comment:
Can we parametrize the query (or at least quote identifiers) https://stackoverflow.com/questions/62396036/is-there-a-difference-between-quote-ident-and-parametized-queries-in-psycopg?

Reply:
I think the most important part is where variables are used, i.e. schema and table in this case.

Reply:
Agree. I will address this. But only in the current function. I opened an issue #604 I will go over all the other functions but not in this PR since small typos can lead to bigger errors and this PR should be mainly a fix for the upcoming stable release. Ok?

Reply:
Still not sure what you mean. You think the using of `{schema}.{table}` is bad?

Reply:
like this?

```
           cur.execute(
                sql.SQL( """SELECT setting
                           FROM {}.{}
                           WHERE tag = %s
                        """).format(sql.Identifier(self.schema),sql.Identifier(PG_SETTINGS_TABLE)), ("ch.ehi.ili2db.TidHandling",)
            )

```

Reply:
> Still not sure what you mean. You think the using of {schema}.{table} is bad? 

Exactly, e.g. if schema name or table name contains spaces or other special characters the query will fail.

What you proposed looks good.
Or:

```
cur.execute( 
"""SELECT setting
                           FROM {schema}.{table}
                           WHERE tag = %s
                        """.format(schema=ext.quote_ident(schema), table=ext.quote_ident(table))
)
```

Reply:
We have sollutions on `psycopg2` - do you know what we can use for `sqlite3` and `pyodbc`?

Reply:
A quick search for sqlite3 didn't reveal any core functionality (some helper methods like https://pretagteam.com/question/sqlite3-escapestring-failing-multiple-single-quotes-generated)
For pyodbc, maybe @gacarrillor has an idea?

Reply:
Can we move the solution finding here #604 and have this PR "in dry"? :-)

Reply:
Yes, that sounds good to me (and was the reason for only commenting on the PG implementation)